scripts/dts/extract/clocks.py

```diff
@@ -42,8 +41,8 @@ def _extract_consumer(self, node_address, yaml, clocks, def_label):
                                     str(clock_provider)))
                 clock_provider_node_address = phandles[cell]
                 clock_provider = reduced[clock_provider_node_address]
-                clock_provider_compat = get_compat(clock_provider_node_address)
-                clock_provider_bindings = yaml[clock_provider_compat]
+                clock_provider_bindings = get_binding(
+                                            clock_provider_node_address)
```

Comment:
Off topic but this sort of thing strengthens my belief that a strict 80-character limit on line length is decreasing code clarity more than increasing it.  Especially when EOL escapes used to meet the requirement aren't consistent (see next line).

Reply:
The 80 char limit is not enforced. `checkpatch` will emit a warning, but CI will not fail. Contributors are free to exceed the limit when it makes sense to do so.